dove/tests/helper.rs

```diff
@@ -0,0 +1,135 @@
+#![allow(dead_code)]
+use std::path::{PathBuf, Path};
+use std::fs::{remove_dir_all, read_to_string, create_dir_all};
+use fs_extra::file::write_all;
+use toml::Value;
+use dove::cli::execute;
+
+pub fn project_start(project_name: &str) -> (PathBuf, PathBuf) {
+    let tmp_folder = std::env::temp_dir();
+    let project_folder = tmp_folder.join(project_name);
+    if project_folder.exists() {
+        remove_dir_all(&project_folder).unwrap();
+    }
+    (tmp_folder, project_folder)
+}
+pub fn project_start_for_init(project_name: &str) -> PathBuf {
```

Comment:
nit: I suggest adding newlines between function definitions. It's pretty weird rustfmt doesn't do it